unit_tests/test_scylla_bench_thread.py

```diff
@@ -76,3 +76,31 @@ def cleanup_thread():
 
     assert not errors
     assert summaries[0]["Clustering row size"] == "Uniform(min=10, max=20)"
+
+
+def test_02_scylla_bench(request, docker_scylla_ssl, params):
```

Comment:
Please, add word "tls" to the test name. It will be useful later.

Reply:
And, in general, this new test and above one are almost complete duplicates. Would be good to have reused code here.